Tests/XMLCoderTests/CombineTests.swift

```diff
@@ -27,6 +27,12 @@ private struct Foo: Codable {
     var name: String
 }
 
+final class CustomEncoder: XMLEncoder {
+    override func encode<T>(_ value: T) throws -> Data where T : Encodable {
```

Comment:
```suggestion
    override func encode<T>(_ value: T) throws -> Data where T: Encodable {
```

Reply:
I'm sorry I didn't check SwiftFormat. I fixed in 18282b5515145500190dd817c9b0eab2b99188ed.